packages/calypso-e2e/src/lib/blocks/block-flows/instagram.ts

```diff
@@ -56,7 +51,11 @@ export class InstagramBlockFlow implements BlockFlow {
 	 * @param {PublishedPostContext} context The current context for the published post at the point of test execution
 	 */
 	async validateAfterPublish( context: PublishedPostContext ): Promise< void > {
-		// await context.page.pause();
-		await context.page.locator( '.wp-block-embed-instagram' ).waitFor();
+		await context.page
+			.getByRole( 'figure' )
+			.filter( {
+				hasText: 'View this post on Instagram',
+			} )
+			.waitFor();
```

Comment:
Would this work as well?
```suggestion
		await context.page
			.getByText( 'View this post on Instagram' )
			.waitFor();
```

Reply:
I don't see why it wouldn't work - let me give it a try.

Reply:
Seems to work locally with proxy on and off. 

Triggering a Gutenberg build in CI to make sure it works there too.